daq/faucet_state_collector.py

```diff
@@ -84,7 +89,14 @@ def get_switches(self):
             switch_data[switch_name] = self.get_switch(switch_name)
         return switch_data
 
-    def get_switch_map(self):
+    def _get_egress_state(self):
+        """Return egress state obj"""
+        with self.lock:
+            egress_obj = self.topo_state.get(EGRESS_STATE, {})
+            return egress_obj.get(EGRESS_STATUS), \
```

Comment:
yah, line breaks are ugly and an indication (smell) that this is wrong.  As per above, just pass in the target object and update it here.